src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -8812,16 +8812,18 @@ Real64 CalcClearRoof(EnergyPlusData &state,
     if (x > 0.0) {
         return CalcClearRoof(state, SurfTemp, AirTemp, WindAtZ, RoofArea, RoofPerimeter, RoughnessIndex);
     } else {
-        if (state.dataConvectionCoefficient->CalcClearRoofErrorIDX == 0) {
-            ShowSevereMessage(state, "CalcClearRoof: Convection model not evaluated (bad value for distance to roof edge)");
-            ShowContinueError(state, format("Value for distance to roof edge ={:.3R}", x));
-            ShowContinueError(state, format("Occurs for surface named = {}", state.dataSurface->Surface(SurfNum).Name));
-            ShowContinueError(state, "Convection surface heat transfer coefficient set to 9.999 [W/m2-K] and the simulation continues");
+        if (state.dataSurface->Surface(SurfNum).ExtBoundCond != DataSurfaces::OtherSideCondModeledExt) {
+            if (state.dataConvectionCoefficient->CalcClearRoofErrorIDX == 0) {
+                ShowSevereMessage(state, "CalcClearRoof: Convection model not evaluated (bad value for distance to roof edge)");
+                ShowContinueError(state, format("Value for distance to roof edge ={:.3R}", x));
+                ShowContinueError(state, format("Occurs for surface named = {}", state.dataSurface->Surface(SurfNum).Name));
+                ShowContinueError(state, "Convection surface heat transfer coefficient set to 9.999 [W/m2-K] and the simulation continues");
+            }
+            ShowRecurringSevereErrorAtEnd(
+                state,
+                "CalcClearRoof: Convection model not evaluated because bad value for distance to roof edge and set to 9.999 [W/m2-k]",
+                state.dataConvectionCoefficient->CalcClearRoofErrorIDX);
```

Comment:
Now the error message is reported only when the exterior boundary condition of a surface is not `OtherSideCondModeledExt` because this boundary condition does not use the convection coefficient calculated here.

Reply:
This fix makes perfect sense, you wouldn't want to emit an error for that case.